manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -24,15 +14,11 @@
       {{- else if eq .Values.global.proxy.tracer "zipkin" }}
         zipkin:
           # Address of the Zipkin collector
-        {{- if .Values.global.tracer.zipkin.address }}
-          address: {{ .Values.global.tracer.zipkin.address }}
-        {{- else }}
-          address: zipkin.{{ .Values.global.telemetryNamespace }}:9411
-        {{- end }}
+          address: {{ .Values.global.tracer.zipkin.address | default "zipkin:9411" }}
```

Comment:
`| defaults` here are so we can have deprecation notices but still set a default